docs/source/deployment/prefect.md

```diff
@@ -12,9 +12,9 @@ Prefect Server ships out-of-the-box with a fully featured user interface.
 
 To use Prefect Core and Prefect Server, make sure you have the following prerequisites in place:
 
-- Prefect Core is [installed](https://docs.prefect.io/core/getting_started/install.html) on your machine
+- [Prefect Core is installed](https://docs.prefect.io/core/getting_started/install.html) on your machine
 - [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/) are installed and Docker Engine is running
-- Prefect Server is [up and running](https://docs.prefect.io/orchestration/Server/deploy-local.html)
+- [Prefect Server is up and running](https://docs.prefect.io/orchestration/Server/deploy-local.html)
 - `PREFECT__LOGGING__EXTRA_LOGGERS` environment variable is set (it is required to get Kedro logs emitted):
```

Comment:
- `PREFECT__LOGGING__EXTRA_LOGGERS` environment variable is set (this is required to get Kedro logs published):